app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListViewModel.kt

```diff
@@ -73,6 +74,8 @@ class TransactionListViewModel
         type: TransactionPagingSource.Type
     ): Flow<PagingData<TransactionView>> {
 
+        val chain = safe.chain!!
```

Comment:
This is only ok, if the safe comes from a database. And it does in this case. But is there a better way. For example: Could we make chain not nullable? And then add the default chain data everytime we create a safe on the fly?